packages/coinjoin/src/enums.ts

```diff
@@ -10,6 +10,7 @@ export enum SessionPhase {
     RetryingRoundPairing = 17,
     AffiliateServerOffline = 18,
     CriticalError = 19,
+    BlockedUtxos = 20,
```

Comment:
I'm afraid this can cause some troubles cc @dahaca  
https://github.com/trezor/trezor-suite/blob/feat/cj-blocked-utxos/packages/suite/src/utils/wallet/coinjoinUtils.ts/#L319

Reply:
not sure, message is displayed properly from what i tested

but i guess this logic needs to be done better at some point